simulator/src/main/java/com/hazelcast/simulator/coordinator/TestCaseRunner.java

```diff
@@ -119,8 +119,8 @@ boolean run() {
             echo("Finished Test global tear down");
 
             echo("Starting Test local tear down");
-            agentsClient.waitForPhaseCompletion(prefix, testCaseId, "localTeardown");
             agentsClient.executeOnAllWorkers(new GenericCommand(testCaseId, "localTeardown"));
+            agentsClient.waitForPhaseCompletion(prefix, testCaseId, "localTeardown");
```

Comment:
how the old version could ever worked? it's waiting for end of the phase which never started?